src/main/java/org/jabref/gui/entryeditor/OtherFieldsTab.java

```diff
@@ -24,19 +24,6 @@ public OtherFieldsTab(BibDatabaseContext databaseContext, SuggestionProviders su
         setGraphic(IconTheme.JabRefIcon.OPTIONAL.getGraphicNode());
     }
 
-    public static boolean isOtherField(EntryType entryType, String fieldToCheck) {
```

Comment:
Any reason why you deleted this?
Otherwise lgtm

Reply:
Yes, the code was no longer used.